msal/application.py

```diff
@@ -79,6 +79,30 @@ def extract_certs(public_cert_content):
     return [public_cert_content.strip()]
 
 
+def _merge_claims_and_capabilities(capabilities, claims):
+    if claims:
+        claims = json.loads(claims)
+    if capabilities:
+        client_capabilities_dict = {
+            "access_token": {
+                "xms_cc": {
+                    "values": capabilities
+                }
+            }
+        }
+        if claims:
+            for key in client_capabilities_dict:
+                claims_node = claims.get(key, {})
+                if claims_node:
+                    claims_node.update(client_capabilities_dict[key])
+                else:
+                    claims[key] = client_capabilities_dict[key]
+
+        else:
+            return json.dumps(client_capabilities_dict)
+    return json.dumps(claims)
```

Comment:
Currently, if claims input is `None`, the output would become `null`. This might not be what we want?

Also, we should have a test case for such edge case.